src/renderer/store/modules/utils.js

```diff
@@ -611,15 +606,13 @@ const actions = {
     externalPlayerMap.unshift(playerNone)
 
     const externalPlayerNames = externalPlayerMap.map((entry) => { return entry.name })
-    const externalPlayerNameTranslationKeys = externalPlayerMap.map((entry) => { return entry.nameTranslationKey })
     const externalPlayerValues = externalPlayerMap.map((entry) => { return entry.value })
```

Comment:
Could you please remove `nameTranslationKey` on line 601 as well (I can't comment on unchanged lines).